src/main/scala/mesosphere/marathon/health/HealthCheckActor.scala

```diff
@@ -51,10 +52,7 @@ class HealthCheckActor(
     )
   }
 
-  // self-sent every healthCheck.intervalSeconds
```

Comment:
OK to move `Tick` to the companion object, but can we carry this comment there as well?